install-dev/init.php

```diff
@@ -82,6 +82,10 @@
 require_once _PS_CORE_DIR_.'/config/autoload.php';
 if (file_exists(_PS_CORE_DIR_.'/app/config/parameters.php')) {
     require_once _PS_CORE_DIR_.'/config/bootstrap.php';
+
+    $kernel = new AppKernel(_PS_MODE_DEV_ ? 'dev' : 'prod', _PS_MODE_DEV_);
```

Comment:
Should you have to check for `test` env?

Reply:
This is what we got on the previous version, but is this case possible during installation?

Reply:
It's just to say to the kernel, "Please use var/cache/test instead of dev :P" and we will able to use specific services for test env.